digid_eherkenning/saml2/eherkenning.py

```diff
@@ -32,6 +35,214 @@
+        "entity_id": eherkenning_config.entity_id,
+        "oin": eherkenning_config.oin,
+        "organisation_name": eherkenning_config.organization_name,
+        "services": [
+            {
+                "attribute_consuming_service_index": eherkenning_config.eh_attribute_consuming_service_index,
+                "service_loa": eherkenning_config.loa,
+                "service_uuid": str(uuid4()),
+                "service_name": {
+                    "nl": eherkenning_config.service_name,
+                    "en": eherkenning_config.service_name,
+                },
+                "service_description": {
+                    "nl": eherkenning_config.service_description,
+                    "en": eherkenning_config.service_description,
+                },
+                "service_instance_uuid": str(uuid4()),
+                "service_url": eherkenning_config.base_url,
+                # Either require and return RSIN and KVKNr (set 1) or require only KvKnr (set 2). The
+                # latter is needed for 'eenmanszaak'
+                "entity_concerned_types_allowed": [
+                    {
+                        "set_number": "1",
+                        "name": "urn:etoegang:1.9:EntityConcernedID:RSIN",
+                    },
+                    {
+                        "set_number": "1",
+                        "name": "urn:etoegang:1.9:EntityConcernedID:KvKnr",
+                    },
+                    {
+                        "set_number": "2",
+                        "name": "urn:etoegang:1.9:EntityConcernedID:KvKnr",
+                    },
+                ],
+                "requested_attributes": [
+                    {
+                        "name": "urn:etoegang:DV:%(oin)s:services:%(index)s"
+                        % {
+                            "oin": eherkenning_config.oin,
+                            "index": eherkenning_config.eh_attribute_consuming_service_index,
+                        },
+                        "isRequired": False,
+                    }
+                ],
+                "privacy_policy_url": {
+                    "nl": eherkenning_config.privacy_policy,
+                },
+                "herkenningsmakelaars_id": eherkenning_config.makelaar_id,
+            },
+            {
+                "attribute_consuming_service_index": eherkenning_config.eidas_attribute_consuming_service_index,
+                "service_loa": eherkenning_config.loa,
+                "service_uuid": str(uuid4()),
+                "service_name": {
+                    "nl": eherkenning_config.service_name + " (eIDAS)",
+                    "en": eherkenning_config.service_name + " (eIDAS)",
+                },
+                "service_description": {
+                    "nl": eherkenning_config.service_description,
+                    "en": eherkenning_config.service_description,
+                },
+                "service_instance_uuid": str(uuid4()),
```

Comment:
does it make sense that this UUID is different every time metadata is generated?